schema/url/visualizations.raml

```diff
@@ -57,15 +57,23 @@ types:
   SampleSize:
     type: SimpleSampleSize | ProportionSampleSize
 
+  # continue to allow deprecated values for now
+  ShowMissingness:
+    type: string
+    enum: ['allVariables', 'strataVariables', 'noVariables', 'TRUE', 'FALSE']
+
+  ShowMissingnessNoAxes:
+    type: string
+    enum: ['strataVariables', 'noVariables', 'TRUE', 'FALSE']
+
   # Assuming computed variable must be on y axis and that x axis may be determined by collection var or specified.
   BoxplotWith1ComputeSpec:
     additionalProperties: false
     properties:
       outputEntityId:
         type: string
       showMissingness:
-        type: string
-        enum: ['TRUE', 'FALSE']
+        type: ShowMissingness
```

Comment:
regular boxplot spec has ShowMissingnessNoAxes so that should probably also be the case here, too?